clusters/build-clusters/01_cluster/openshift/ci-operator/registry-pull-credentials_secret_template.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+objects:
+- apiVersion: v1
+  kind: Secret
+  metadata:
+    annotations:
+      openshift.io/description: |
+        This secret contains the pull secrets for api.ci cluster and the local registry.
+        It is used to pull images 
+        * by the image clients in ci-operator pod: the local images synced by dptp-controller are eventually consistent only
+        * by the build pods created by ci-operator: https://bugzilla.redhat.com/show_bug.cgi?id=1861546
+
+    name: registry-pull-credentials
+    namespace: ${namespace}
+  stringData:
+    .dockerconfigjson: |-
+      {
+        "auths": {
+          "registry.svc.ci.openshift.org": {
+            "auth": "${build01_ci_reg_auth_value}"
+          },
+          "default-route-openshift-image-registry.apps.build01.ci.devcluster.openshift.com": {
```

Comment:
Can we have just one template and make the clutername a variable to prevent drift across clusters?

Reply:
I'm also wondering why the build01 is different and has this record (the others do not have a record for their external registry URLs). But I'd be careful about changing too much in what is essentially a "rename secret" task.

Reply:
The external registry url was removed in https://github.com/openshift/release/pull/12093

Reply:
So we should remove this as well from this template, right?

Reply:
I remove it from the template in #12093

Reply:
removed here as well.